### PR TITLE
Merge master to dev

### PR DESCRIPTION
**Pull request by kreedula**

### CODE REVIEW

File: changelog [hunk 1]
```diff
@@ -1,5 +1,12 @@
 MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki
 
+Version 1.4.1
+- Hotfix release for Teams Dog Food.
```
Comment (rpdome) [inline changelog:4]: nit: not sure if DogFood is a customer-facing term 😆

File: msal/build.gradle [hunk 2]
```diff
@@ -145,9 +145,9 @@ dependencies {
         transitive = false
     }
 
-    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '2.0.3-SNAPSHOT', changing: true)
+    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '2.0.6-SNAPSHOT', changing: true)
 
-    distApi("com.microsoft.identity:common:2.0.2") {
+    distApi("com.microsoft.identity:common:2.0.2-hf1") {
```
Comment (rpdome) [inline msal/build.gradle:150]: should this be 2.0.6?

Comment (rpdome) [reply msal/build.gradle:150]: Yes, but this is being merged into dev no?

Comment (shahzaibj) [reply msal/build.gradle:150]: Yes, this is being merged into dev

Comment (kreedula) [reply msal/build.gradle:150]: dist API version of MSAL in `dev` is `1.4.1` which depends on `2.0.2-hf1` , `2.0.6` was for `broker`

**Pull request merged**